TutorLizard.Web/Views/Shared/_AdRequests.cshtml

```diff
@@ -1,8 +1,9 @@
 ﻿@model AdRequestsListDto
```

Comment:
Could you make styles in this partial the same as in `_AdListItem` and add a button like the one there?